### PR TITLE
Deprecations will be done biannually (Jan and July).

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

Deprecation of modules/argspec keys will be done either in January or July of the specified deprecation year.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

File: changelogs/fragments/deprecation_biannually.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```
Comment (NilashishC) [inline changelogs/fragments/deprecation_biannually.yaml:2]:
```suggestion
trivial:
```

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**